test/extensione2e/kafka/kafka_source_to_ksvc_test.go

```diff
@@ -37,6 +37,7 @@ const (
 )
 
 var (
+	// TODO: TLS/SASL? --> https://issues.redhat.com/browse/SRVKE-647
```

Comment:
Just noticed this and created https://issues.redhat.com/browse/SRVKE-647.
As I wrote in the PR description, TLS/SASL testing for KafkaSource is not part of this PR.

KafkaSource has the ability to receive the auth config in its spec. It won't use the auth config in `config-kafka`, which is what we modify part of this PR.

Reply:
See also https://issues.redhat.com/browse/SRVKE-651  that @lberk  is currently assigned to